css/colorVariables.css

```diff
@@ -0,0 +1,40 @@
+:root {
+    --navyBlue: #082032;
```

Comment:
I suggest not to include the variable value in the variable name.

Reason - If in future you decide to change navyBlue to say purple, you'd have to change the references to this variable wherever you used. 
Please choose a generic enough name.